Libraries/Components/WebView/WebView.ios.js

```diff
@@ -118,6 +119,20 @@ var WebView = React.createClass({
      */
     bounces: PropTypes.bool,
     /**
+     * A floating-point number that determines how quickly the scroll view
+     * decelerates after the user lifts their finger. You may also use string
+     * shortcuts `"normal"` and `"fast"` which match the underlying iOS settings
+     * for `UIScrollViewDecelerationRateNormal` and
+     * `UIScrollViewDecelerationRateFast` respectively.
+     *   - Normal: 0.998
+     *   - Fast: 0.9 (the default for iOS WebView)
+     * @platform ios
+     */
+    decelerationRate: PropTypes.oneOfType([
```

Comment:
Since you're importing ScrollView.js anyway, you can avoid the repetition here by defining this as:

    decelerationRate: ScrollView.propTypes.decelerationRate,